src/DynamoRevit/DynamoRevit.cs

```diff
@@ -152,6 +152,13 @@ public class JournalKeys
         public const string ModelShutDownKey = "dynModelShutDown";
 
         /// <summary>
+        /// The journal file can specify if a check should be performed to see if the
+        /// existing model already points to the Dynamo file we want to run or perform
+        /// other tasks. If that's the case, we want to use the existing model.
```

Comment:
can you clarify this comment a bit, when you refer to Model, please specify workspaceModel vs DynamoModel